mkdocs.yml

```diff
@@ -110,6 +110,7 @@ markdown_extensions:
 
 nav: # This lists all the files that become part of the documentation
   - Home: 'index.md'
+  - A introduction to Jujutsu: 'jj_first_principles.md'
```

Comment:
I added this at the toplevel to partially answer questions like https://bsky.app/profile/odbol.bsky.social/post/3lerkmjjmic25